packages/open-payments/src/test/helpers.ts

```diff
@@ -91,6 +91,8 @@ export const mockOutgoingPayment = (
   },
   quoteId: uuid(),
```

Comment:
`quoteId` and `id` are supposed to be `uri`s
https://github.com/interledger/open-payments/blob/02a6d505d79124cce776054df89e28f02c8e5d59/openapi/resource-server.yaml#L1072-L1084
Am I correct that we shouldn't be concerned about the quality of our OpenAPI validation because this package is only testing against `successfulValidator`/`failedValidator`?

Reply:
Good point, I'll make the test case a bit more realistic. But yes, in general, IMO the actual OpenAPI spec validation is a bit out of scope of the package, that responsibility is `openapi`s